istioctl/cmd/revision.go

```diff
@@ -203,12 +205,16 @@ func revisionList(writer io.Writer, args *revisionArgs, logger clog.Logger) erro
 		if ri := revisions[rev]; ri == nil {
 			revisions[rev] = &tag.RevisionDescription{}
 		}
+		cs, err := getEnabledComponents(iop)
+		if err != nil {
+			return fmt.Errorf("error while get IstioOperator Components: %v", err)
```

Comment:
```suggestion
			return fmt.Errorf("error while getting IstioOperator Components: %v", err)
```
nit: can you improve some of the error messsages to be more clearly?